css/main.css

```diff
@@ -236,10 +261,79 @@
     margin-bottom: 34px;
   }
 
+  .features {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .features-container .feature__item {
+    display: flex;
+    gap: 20px;
+    box-shadow: rgba(0, 0, 0, 0.2) 0 0 3px;
+    margin: 16px;
+    padding: 10px;
+    border-radius: 5px;
+    transition: all 0.5s ease;
+  }
+
+  .features-container .last {
+    display: none;
+    transition: all 0.5s ease;
+  }
+
+  .features-container .last.active {
+    display: flex;
+    transition: all 0.5s ease;
+  }
+
+  .feature__item .title {
+    color: #fd6b32;
+  }
+
+  .feature__item .text {
+    width: auto;
+    margin-top: 15px;
+  }
+
+  .features button {
+    background: none;
+    border: 1px solid #e6e6e3;
+    width: 94%;
+    padding: 1rem 0;
+    margin-bottom: 50px;
+    font-size: 1.5rem;
+    border-radius: 0.3rem;
+    text-align: initial;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    transition: all 0.5s ease;
+  }
+
+  .features button i {
+    font-size: 34px;
+    margin-top: 5px;
+    color: #fd6b32;
+  }
+
+  #header nav button {
+    position: fixed;
+    left: 24px;
+    top: 10px;
+    border: none;
+    font-size: 34px;
+    cursor: pointer;
+    transition: all 0.3s ease;
+    background: transparent;
+    border-radius: 10px;
+    height: 40px;
+  }
+
   .about {
     text-align: center;
-
```

Comment:
As said previously 
- [ ] Kindly make sure this text is well-centered and not overcrowded as it is on your mobile version
`hint: You may consider giving its container a width and margin: ...px auto and then remove the align: center or if you need the align center you may consider giving its container a consistent width. eg: 80% or even 100%`
![Screenshot from 2022-10-19 19-06-45](https://user-images.githubusercontent.com/97100091/196758534-d247b219-29d0-47b5-99ef-53f8ece535ef.png)